bindings/rust/Cargo.toml

```diff
@@ -12,6 +12,9 @@ description = "Bindings to Fizzy"
 categories = ["external-ffi-bindings"]
 edition = "2018"
 
+[dependencies]
+hex = "0.4.2"
```

Comment:
What kind of dependency is it? This is only required for tests.

Reply:
Yeah, will need to move it do `dev-dependencies`, forgot to do it last week.

Reply:
Done.